### PR TITLE
Only run test with ffmpeg 4.2

### PR DESCRIPTION
**Pull request by tianjunwork**
ffmpeg plugin patch can't apply to ffmpeg master for now.
Just run test for ffmpeg 4.2 release branch.

Signed-off-by: Jun Tian <jun.tian@intel.com>

### CODE REVIEW

**Comment by tianjunwork** [comment]
Hi @1480c1, could we just run test on ffmpeg release branch at least for now? Maybe re-enable master after next ffmpeg release when we don't have conflict applying patch on both master and release branch.

**Comment by 1480c1** [comment]
Looks good, although you may want to consider downloading zip archive and using patch instead to remove the git repo overhead

File: appveyor.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ for:
     build_script:
       - make -C Build -j install
       - pkg-config --debug --exists --print-errors SvtVp9Enc
-      - git clone https://github.com/FFmpeg/FFmpeg.git --depth=1 --branch=master ffmpeg
+      - git clone https://github.com/FFmpeg/FFmpeg.git --depth=1 --branch=release/4.2 ffmpeg
```
Comment (1480c1) [inline appveyor.yml]:
```suggestion
      - Invoke-WebRequest -UseBasicParsing -OutFile ffmpeg.zip -Uri "https://github.com/FFmpeg/FFmpeg/archive/release/4.2.zip"
      - 7z x -aoa -offmpeg ffmpeg.zip
      - Get-ChildItem ffmpeg/FFmpeg-*/* | ForEach-Object {Move-Item $_.FullName ffmpeg}
```

File: appveyor.yml [hunk 2]
```diff
@@ -39,7 +39,9 @@ for:
     build_script:
       - make -C Build -j install
       - pkg-config --debug --exists --print-errors SvtVp9Enc
-      - git clone https://github.com/FFmpeg/FFmpeg.git --depth=1 --branch=master ffmpeg
+      - Invoke-WebRequest -UseBasicParsing -OutFile ffmpeg.zip -Uri "https://github.com/FFmpeg/FFmpeg/archive/release/4.2.zip"
+      - 7z x -aoa -offmpeg ffmpeg.zip
+      - Get-ChildItem ffmpeg/FFmpeg-*/* | ForEach-Object {Move-Item $_.FullName ffmpeg}
```
Comment (1480c1) [inline appveyor.yml]:
```suggestion
      - ps: Invoke-WebRequest -UseBasicParsing -OutFile ffmpeg.zip -Uri "https://github.com/FFmpeg/FFmpeg/archive/release/4.2.zip"
      - 7z x -aoa -offmpeg ffmpeg.zip
      - ps: Get-ChildItem ffmpeg/FFmpeg-*/* | ForEach-Object {Move-Item $_.FullName ffmpeg}
```

Comment (tianjunwork) [reply appveyor.yml]: Thank you:)

File: appveyor.yml [hunk 3]
```diff
@@ -39,7 +39,9 @@ for:
     build_script:
       - make -C Build -j install
       - pkg-config --debug --exists --print-errors SvtVp9Enc
-      - git clone https://github.com/FFmpeg/FFmpeg.git --depth=1 --branch=master ffmpeg
+      - ps: Invoke-WebRequest -UseBasicParsing -OutFile ffmpeg.zip -Uri "https://github.com/FFmpeg/FFmpeg/archive/release/4.2.zip"
+      - 7z x -aoa -offmpeg ffmpeg.zip
+      - ps: Get-ChildItem ffmpeg/FFmpeg-*/* | ForEach-Object {Move-Item $_.FullName ffmpeg}
       - cd ffmpeg
       - git apply --ignore-space-change --ignore-whitespace %APPVEYOR_BUILD_FOLDER%/ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch
```
Comment (1480c1) [inline appveyor.yml]:
```suggestion
      - patch --binary -s -N -p1 -i "%APPVEYOR_BUILD_FOLDER%/ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch"
```

I'm not sure if git apply with still patch the files if it's not git

**Pull request merged**